app/javascript/dashboard/modules/widget-preview/components/WidgetBody.vue

```diff
@@ -9,7 +9,7 @@
           {{ config.replyTime }}
         </div>
       </div>
-      <thumbnail username="J" size="40" />
+      <thumbnail username="J" size="40px" />
```

Comment:
What does this J do?

Reply:
This is a random name shown on the preview
<img width="417" alt="Screenshot 2022-11-15 at 5 06 52 PM" src="https://user-images.githubusercontent.com/2246121/202058252-418e57b1-d2b6-438f-bf11-ff03206950f3.png">